src/lib/select/option/option-foundation.ts

```diff
@@ -24,12 +24,13 @@ export class OptionFoundation implements IOptionFoundation {
   constructor(private _adapter: IOptionAdapter) {}
 
   /** Gets/sets the value of this option. */
-  public get value(): string {
+  public get value(): any {
```

Comment:
We should probably transition to `unknown` at some point

Reply:
Agreed. I'll add that to the 3.0 roadmap. This was just a mismatching type because everywhere else that interfaces with this uses `any` currently...